examples/llm_serving/README.md

```diff
@@ -0,0 +1,50 @@
+# Run your own LLM chatbot on the cloud in 4 easy steps!
+
+[FlexGen](https://github.com/FMInference/FlexGen) is a high-throughput generation engine for running large language models (LLMs) with limited GPU memory (e.g., a 16GB T4 GPU or a 24GB RTX3090 gaming card!)
```

Comment:
Since FlexGen removed the chatbot from their repo, can we change the LLM model from the huggingface instead? 
We will submit a PR to the FlexGen repo with the yaml for benchmark, but in our repo, can we have a `chatbot.py` in our own repo which loads the huggingface model.